### PR TITLE
Fake data generation

### PR DESCRIPTION
**Pull request by lytonnyemba**
I have added the relationship between facility, district and zone and i have also created remote method for downloading all three files namely; pdf, excel and csv and i have edited script to be able to generate 10 dependencies all pivot tables

### CODE REVIEW

File: common/models/facility.js [hunk 1]
```diff
@@ -1,5 +1,266 @@
...
+						footer: function(currentPage, pageCount) {
+							return {
+								margin: 10,
+								columns: [
+									{
+										fontSize: 9,
+										text: [
+											{
+												text:
+"--------------------------------------------------------------------------" +
```
Comment (liwoo) [inline common/models/facility.js]: You should repeat this programmatically

File: common/models/facility.js [hunk 2]
```diff
@@ -1,5 +1,266 @@
...
 
+	Facility.remoteMethod("download", {
+		description: "Download all facilities of your choice",
+		accepts: [
+			{arg: "data", type: "object"},
+			{arg: "res", type: "object", http: {source: "res"}}
+		],
+		returns: {}
+	});
+	Facility.downloadOne = function(inputData, res, cb) {
```
Comment (liwoo) [inline common/models/facility.js]: Find out in the loopback documentation if you can have two remote methods of the same model in two separate files

File: common/models/facility.js [hunk 3]
```diff
@@ -1,5 +1,266 @@
-'use strict';
+"use strict";
 
+var PdfPrinter = require("pdfmake");
```
Comment (liwoo) [inline common/models/facility.js]: make these constants

**Comment by lytonnyemba** [comment]
I have added common name field in facility model and village, ta in physical address model

**Pull request merged**